cypress-tests/cypress/e2e/PaymentUtils/Cybersource.js

```diff
@@ -189,6 +189,23 @@ export const connectorDetails = {
         currency: "USD",
         customer_acceptance: null,
         setup_future_usage: "on_session",
+        billing: {
+          address: {
+            line1: "1467",
+            line2: "Harrison Street\nApt 101",
+            line3: "Harrison Street\nApt 101",
+            city: "San Fransico\n city",
+            state: "California",
+            zip: "94122",
+            country: "NL",
+            first_name: "joseph",
+            last_name: "Doe",
+          },
+          phone: {
+            number: "9123456789",
+            country_code: "+91",
+          },
+        },
```

Comment:
keeping re-usability in mind, can we move this to a `const` and use it here? that way, if tomorrow there arises a need for passing another object of billing, we can directly pass the constant instead.